satellite/overlay/service.go

```diff
@@ -239,6 +252,11 @@ func (service *Service) IsOnline(node *NodeDossier) bool {
 	return time.Since(node.Reputation.LastContactSuccess) < service.config.Node.OnlineWindow
 }
 
+// IsOnline checks if a node is 'online' based on the collected statistics.
+func (service *Service) IsNew(node *NodeDossier) bool {
+	return time.Since(node.Reputation.LastContactSuccess) < service.config.Node.OnlineWindow
```

Comment:
@mobyvb could you tell what I need to check here

Reply:
You need access to a NodeSelectionConfig (see FindStorageNodeWithPreferences below). The values are configured above the dB layer in satellite/overlay. Then you check that the total audit count and total uptime count are less than the values in that struct
At an appointment right now but can talk more tomorrow.